hasura.planx.uk/metadata/tables.yaml

```diff
@@ -798,11 +812,20 @@
           - version
         filter:
           _and:
-            - team_id:
-                _eq: 32
-            - creator_id:
-                _eq: x-hasura-user-id
-        check: null
+            - _and:
+                - team_id:
+                    _eq: 32
+                - creator_id:
+                    _eq: x-hasura-user-id
+            - deleted_at:
+                _is_null: true
+        check:
+          _and:
+            - deleted_at:
+                _is_null: false
+            - templated_flows:
+                id:
+                  _is_null: false
```

Comment:
This repeated pattern is a post-update check which ensures that flows cannot be archived if they have associated templated flows derived from them.